todoapp/src/components/TodoList.vue

```diff
@@ -0,0 +1,92 @@
+<script setup lang="ts">
+import { onMounted, ref, watch } from 'vue'
+import TodoForm from './TodoForm.vue'
+import Header from './Header.vue'
+import Todos from './Todos.vue'
+
+const todos = ref<{ text: string }[]>([])
+const inputContent = ref('')
+
+function addTodo(inputContent: string) {
+  if (inputContent.trim() === '') {
+    return
+  }
+
+  todos.value.push({
+    text: inputContent
+  })
+  inputContent = ''
```

Comment:
If you empty the input content in the child component you do not need to do it here as well, after you have done the writable computed